src/m365/pa/commands/app/app-permission-remove.ts

```diff
@@ -0,0 +1,191 @@
+import { Group } from '@microsoft/microsoft-graph-types';
+import { Logger } from '../../../../cli/Logger';
+import GlobalOptions from '../../../../GlobalOptions';
+import request, { CliRequestOptions } from '../../../../request';
+import { aadGroup } from '../../../../utils/aadGroup';
+import { validation } from '../../../../utils/validation';
+import PowerAppsCommand from '../../../base/PowerAppsCommand';
+import commands from '../../commands';
+import { aadUser } from '../../../../utils/aadUser';
+import { accessToken } from '../../../../utils/accessToken';
+import Auth from '../../../../Auth';
+import { Cli } from '../../../../cli/Cli';
+
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  appName: string;
+  userId?: string;
+  userName?: string;
+  groupId?: string;
+  groupName?: string;
+  tenant?: boolean;
+  asAdmin?: boolean;
+  environmentName?: string;
+  confirm?: boolean;
+}
+
+class PaAppPermissionRemoveCommand extends PowerAppsCommand {
+  public get name(): string {
+    return commands.APP_PERMISSION_REMOVE;
+  }
+
+  public get description(): string {
+    return 'Removes permissions to a Power Apps app';
+  }
+
+  constructor() {
+    super();
+
+    this.#initTelemetry();
+    this.#initOptions();
+    this.#initValidators();
+    this.#initOptionSets();
```

Comment:
Let's also assign `string` as Minimist type for `groupName`.

Reply:
No problem! Could you care to explain why we have to add `string` as Minimist type for `groupName`? This will help me understand why we should use this in other cases

Reply:
Certainly,

We use Minimist to parse all arguments. By default, Minimist decides what the type is. Imagine we have a group name called `123`. Minimist will parse it as a number instead of a string and this can result in some trouble.
For this command, it will probably still work because we don't do much logic with `groupName`. However, it's best practice to ensure that this option will always parsed as string, which is not the case now if your group name consists of only numbers.

We don't have to do this for `userName` or `groupId` because they always contain non numeric characters (`-` and `@`).

Reply:
Thank you, that's clear 👍